perfmetrics/scripts/ml_tests/pytorch/run_model.sh

```diff
@@ -46,10 +47,14 @@ echo "logging:
           max-file-size-mb: 1024
           backup-file-count: 3
           compress: true
+metadata-cache:
+  ttl-secs: 1728000
+  stat-cache-max-size-mb: 3200
        " > /tmp/gcsfuse_config.yaml
-nohup /pytorch_dino/gcsfuse/gcsfuse --foreground --type-cache-ttl=1728000s \
-        --stat-cache-ttl=1728000s \
-        --stat-cache-capacity=1320000 \
+
+echo "Created config-file at "$config_filename
+echo "Mounting GCSFuse..."
+nohup /pytorch_dino/gcsfuse/gcsfuse --foreground \
```

Comment:
@Tulsishah could you please check that this file change doesn't break the Pytorch v1.12 amd v2.0 support for long-running tests ?

Reply:
Previously --stat-cache-capacity was 1320000bytes you changed it to 3200mb,  I am not sure about this change.

Reply:
@Tulsishah this is because we've deprecated `stat-cache-capacity` which is in terms of number of entries. Each stat-cache entry is about 2500 bytes, so total 1320000 * 2500 bytes ~= 32000 MB, so I've now replaced it with stat-cache-max-size-mb = 3200.

Reply:
Got it thanks.